config/initializers/content_security_policy.rb

```diff
@@ -51,3 +51,7 @@ def host_to_url(str)
   p.script_src :self, :unsafe_inline, assets_host
   p.style_src  :self, :unsafe_inline, assets_host
 end
+
+AboutController.content_security_policy do |p|
+  p.style_src :self, :unsafe_inline, assets_host
+end
```

Comment:
I am not sure I agree with those changes, but I think the build failures would be avoided by putting this directive directly in `app/controllers/about_controller.rb`

Reply:
It is failed by `JsonldHelper`
https://circleci.com/gh/tootsuite/mastodon/134916?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link

Reply:
Yeah, I think that failure will be avoided if you move that rule out of this file and in `app/controllers/about_controller.rb`.

My understanding is that `JsonLdHelper` works as is thanks to the rules added in `config/initializers/inflections.rb` but that gets loaded *after* `config/initializers/content_security_policy.rb`.

Reply:
But i think it's better when gather all CSP into one file and fix #13874

Reply:
having all CSP rules in one place kind of makes sense, but also having them in more specific places does have sense too (`app/controllers/statuses_controller.rb` already does that for the embed endpoint)

Reply:
It seems not inheritable from original CSP config.. and `assets_host` is the problem.